dev-client/src/constants.ts

```diff
@@ -33,5 +33,6 @@ export const FORM_LABEL_MAX = 10;
 export const USER_ROLES = ['viewer', 'contributor', 'manager'];
 export const PROJECT_DEFAULT_MEASUREMENT_UNITS = 'METRIC';
 export const MEASUREMENT_UNITS = ['METRIC', 'IMPERIAL'];
+export const USER_DISTANCE_CONTEXT_CACHE = 5;
```

Comment:
question: what does this mean? (perhaps a more descriptive name will help.)

Reply:
It is the distance in meters the user has to travel until we update the distance sorting context.